### PR TITLE
Server 3.0 PDFs

### PR DESCRIPTION
**Pull request by annabaker**
- Updated script to generate 3.x pdfs 
- Adding PDFs to the sidebar
- Changed xrefs to external docs to hyperlinks so they would continue working in PDFs
- serverversion variable has changed to 3.0.0, so I removed references to this variable in 2.x docs and replaced them with hardcoded versions
- Converted one of the markdowns (private-images.md) to adoc, as we needed it for a server 3 pdf

### CODE REVIEW

File: jekyll/_cci2/aws.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ version:
 :toc: macro
 :toc-title:
 
-Following is a step by step guide to installing CircleCI Server v{serverversion} with Terraform.
+Following is a step by step guide to installing CircleCI Server v2.x with Terraform.
```
Comment (thomasr8) [inline jekyll/_cci2/aws.adoc:13]: Should this be Server v3.x?

File: jekyll/_cci2/overview.adoc [hunk 2]
```diff
@@ -10,7 +10,7 @@ version:
 :toc: macro
 :toc-title:
 
-NOTE: CircleCI Server v{serverversion} uses the CircleCI 2.0 architecture. +
+NOTE: CircleCI Server v2.x uses the CircleCI 2.0 architecture. +
```
Comment (thomasr8) [inline jekyll/_cci2/overview.adoc:13]: Should this be Server v3.x?

File: jekyll/_cci2/private-images.adoc [hunk 3]
```diff
@@ -105,23 +113,24 @@ jobs:
...
-### AWS ECR
+## AWS ECR
 
 CircleCI now supports pulling private images from Amazon's ECR service.
 
-<div class="alert alert-info" role="alert">
-<b>Tip:</b> You can pull your private images from ECR repositories in any regions. However, for the best experience, we strongly recommend to make a copy of your image in <code class="highlighter-rouge">us-east-1</code> region, and specify that <code class="highlighter-rouge">us-east-1</code> image for Docker executor. Our job execution infrastructure is in <code class="highlighter-rouge">us-east-1</code> region so using <code class="highlighter-rouge">us-east-1</code> images makes the <code class="highlighter-rouge"> Spin Up Environement</code> step faster.
-</div>
+TIP: You can pull your private images from ECR repositories in any regions. However, for the best experience, we strongly
+recommend to make a copy of your image in `us-east-1` region, and specify that `us-east-1` image for the Docker executor.
```
Comment (thomasr8) [inline jekyll/_cci2/private-images.adoc:123]:
```suggestion
recommend you make a copy of your image in `us-east-1` region, and specify that `us-east-1` image for the Docker executor.
```

File: jekyll/_cci2/server-ports.adoc [hunk 4]
```diff
@@ -10,7 +10,7 @@ version:
 :toc: macro
 :toc-title:
 
-This section defines the system and port access requirements for installing CircleCI v{serverversion}.
+This section defines the system and port access requirements for installing CircleCI v2.x.
```
Comment (thomasr8) [inline jekyll/_cci2/server-ports.adoc:13]: Should this be v3.x?

**Review by thomasr8 — APPROVED** [review]
Hi Anna....this looks good. I made a few comments about Server version since I was not sure if they should be v2.x or v3.x. Disregard if I am mistaken; I am not sure which it should be. Otherwise, looks really good!

File: jekyll/_cci2/aws.adoc (hunk 1 above)
Comment (annabaker) [reply jekyll/_cci2/aws.adoc:13]: Nope, this is a 2.x doc, double checking your other comments but yeah, a lot of these files weren't named clearly. Maybe tomorrow I'll take some time to refactor the 2.x docs to match new server 3 docs nomenclature

**Pull request merged**